.circleci/config.yml

```diff
@@ -95,6 +95,33 @@ jobs:
           name: Check Javadoc correctness
           command: ./gradlew --max-workers=4 javadoc
 
+      - when:
```

Comment:
All stuff already on `develop`

Reply:
ok, let me update with develop by "Resolve conflicts" / "Update branch" my changes will be tested, ok?
<img width="868" alt="image" src="https://user-images.githubusercontent.com/49597713/187187218-bb9e718a-30c3-4a19-9fb9-e630f5e05bcb.png">